images/win/scripts/ImageHelpers/VisualStudioHelpers.ps1

```diff
@@ -61,20 +61,42 @@ Function Install-VisualStudio
 }
 
 function Get-VsCatalogJsonPath {
-    $instanceFolder = Get-Item "C:\ProgramData\Microsoft\VisualStudio\Packages\_Instances\*" | Select-Object -First 1
-    return Join-Path $instanceFolder.FullName "catalog.json"
+    $instanceFolder = "C:\ProgramData\Microsoft\VisualStudio\Packages\_Instances\" + (Get-VisualStudioInstallation -VSInstallType "VS").InstanceId
+    return Join-Path $instanceFolder "catalog.json"
 }
 
-function Get-VisualStudioPath {
-    return (Get-VSSetupInstance | Select-VSSetupInstance -Product *).InstallationPath
-}
+function Get-VisualStudioInstallation {
+    Param
+    (
+        [Parameter(Mandatory)]
+        [ValidateSet('VS','BuildTools')]
```

Comment:
May be full name for VS to be consistent with BuildTools:`[ValidateSet('VisualStudio','BuildTools')]`

Reply:
Also, I suggest to rename `VSInstallType` to `ProductType`. Here and in other functions.